deploy/sre-prometheus/ocm-agent/100-customer-webooks.PrometheusRule.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: aws-efs-alerts
+    role: alert-rules
+  name: sre-managed-notification-alerts-for-customer-webhooks
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: customer-webhooks
+    rules:
+    - alert: SlowCustomerWebhookSRE
+      expr: |-
+        (
+          label_replace(
+            label_replace(
+              increase(apiserver_admission_webhook_admission_duration_seconds_sum[5m]) /
+                increase(apiserver_admission_webhook_admission_duration_seconds_count[5m]),
+            "type", "mutating", "type", "admit"),
+          "webhook_name", "$0", "name", ".*")
+        ) * on(webhook_name) group_left(service_namespace, service_name) (
+          kube_validatingwebhookconfiguration_webhook_clientconfig_service{service_namespace !~ "openshift-.*"} or
+          kube_mutatingwebhookconfiguration_webhook_clientconfig_service{service_namespace !~ "openshift-.*"}
+        ) > 1
+      for: 15m
+      labels:
+        severity: critical
```

Comment:
Sounds not quite critical to us?

Reply:
Well slow webhook can compromise the operability of the whole cluster.
But I am ok to decrease the severity.

Reply:
Lowering the severity as discussed